packages/trader/src/AppV2/Containers/Trade/trade-types.tsx

```diff
@@ -246,11 +246,14 @@ const TradeTypes = ({ contract_type, onTradeTypeSelect, trade_types }: TTradeTyp
                 </Chip.Selectable>
             ))}
             {should_show_view_all && (
-                <button key='trade-types-all' onClick={handleOpenActionSheet} className='trade__trade-types-header'>
-                    <Text size='sm' bold underlined>
-                        {<Localize i18n_default_text='View all' />}
-                    </Text>
-                </button>
+                <Button
+                    variant='tertiary'
+                    key='trade-types-all'
+                    onClick={handleOpenActionSheet}
+                    className='trade__trade-types-header'
+                    color='black-white'
```

Comment:
This color naming makes my eyes bleed ☠️

Reply:
Agree, but we are working with what we have